### PR TITLE
Update assistants-api-in-a-box (new PR in response to PR #46) 

### LINKED ISSUE #55: Broken links in Assistants API in-a-box README for Sample Assistants
### Operating System

Linux

### Version Information

Python: 3.11.7

### Steps to reproduce

1. Go to /azureai-samples/scenarios/Assistants/assistants-api-in-a-box/notebooks/README.md
2. Scroll to #sample-assistants---jupyter-notebooks section: [sample assistants section](https://github.com/Azure-Samples/azureai-samples/tree/main/scenarios/Assistants/assistants-api-in-a-box#sample-assistants---jupyter-notebooks)
3. Click/follow on any of the 4 sample assistants listed

### Expected behavior

Links should go to each of the 4 sample assistants mentioned

### Actual behavior

File not found for each

### Addition information

⬇️ markdown and relative link issue - should repo in any envio. inc. just browsing readme in github.com 

There are some examples in the /notebooks folder but they're named differently to the ones in the README. It's not clear which is which, or if these are the same samples referred to in the README

🙏 Thank you for providing these samples and READMEs - this is an exciting new feature and this is a great way to get hands-on. keen to test!😄

### PR DESCRIPTION
**Pull request by kdestin**
# Description

Force pushing to #50 irreversibly closed the PR, this PR contains the same changes and some fixes that would've been brought up in review.

Resolves #55 

# Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb [hunk 1]
```diff
@@ -4,9 +4,57 @@
...
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb:20]: Needs objective + link to documentation

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb [hunk 2]
```diff
@@ -4,9 +4,57 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb:39]: Needs a description

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb [hunk 3]
```diff
@@ -4,9 +4,57 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb:39]: Needs a description

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb [hunk 4]
```diff
@@ -4,9 +4,57 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb:39]: Needs a description

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb [hunk 5]
```diff
@@ -4,15 +4,63 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
+   ]
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb:37]: Needs a description.

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb [hunk 6]
```diff
@@ -4,15 +4,63 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
+    "\n",
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n",
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb:20]: Needs objective + link to documentation

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb [hunk 7]
```diff
@@ -4,9 +4,57 @@
...
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb:20]: Needs objective + link to documentation

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb [hunk 8]
```diff
@@ -4,9 +4,57 @@
...
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
```
Comment (kdestin) [inline scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb:20]: Needs objective + link to documentation

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb (hunk 6 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb:20]: # Math Tutor Assistant

**Objective**

This notebook demonstrates the following:

- Showcases the foundational concepts of Assistants such as Threads, Messages, Runs, Tools, and lifecycle management.

This tutorial uses the following Azure AI services:

- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)\n",
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:

- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)
- [Github-OpenAI Python/examples/Assistant demo notebook](https://github.com/openai/openai-python/blob/main/examples/assistant.py)

## Time

You should expect to spend 5-10 minutes running this sample. 

## About this example

The objective of the provided Python file is to create an Azure OpenAI Assistant named "Math Tutor" using the Azure OpenAI API. The assistant is designed to function as a personal math tutor, capable of answering math questions through code interpretation. The script initiates a conversation with the assistant, guiding it through various mathematical queries and scenarios to showcase its capabilities.

This sample provides developers with a clear demonstration of how to integrate the Assistants API into their projects, highlighting its simplicity and effectiveness in leveraging foundational concepts.

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb (hunk 1 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb:20]: # Failed Banks Information Assistant

## Objective

This notebook demonstrates the following:

- Using Assistant tools Code Interpreter and Function calling, this bot can get a CSV file, gather a list of failed banks by state, and generate a chart to visually represent the data.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 5-10 minutes running this sample.

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb (hunk 2 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb:39]: ## About this example
This sample demonstrates the creation of an Azure OpenAI Assistant named "Failed Banks Assistant" through the Azure OpenAI API. The assistant is tailored to aid users in retrieving information about failed banks from a CSV file, the assistant efficiently processes user messages, interacts with the designated file, and promptly delivers pertinent information in response.

### Data
This sample uses files from the folder [`data/banking/`](../data/banking/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb (hunk 3 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb:39]: # Personal Financial Assistant

## Objective
This notebook demonstrates the following:

- Function Calling with Yfinance to get latest stock prices. Summarization of user provided article. Extract country info from article, extract country, capital and other aspects, and call an API to get more information about each country.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant)
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 10-15 minutes running this sample.

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb (hunk 4 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb:39]: # Wind Turbine Farm Management Assistant

## Objective
This notebook demonstrates the following:

- Utilizing Assistant tools such as the Code Interpreter and Function calling, this bot is capable of retrieving a CSV file that illustrates turbine wind speed, voltage, and the last maintenance date. It assists you in reviewing the file contents and aids in determining whether a specific turbine is in need of maintenance.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 5-10 minutes running this sample. 

## About this example
This sample demonstrates the creation of an Azure OpenAI Assistant named "Turbine Management Assistant" utilizing the Azure OpenAI API. The assistant functions as a management tool for wind turbine farms, providing data, charts, and actionable insights regarding the status and performance of each turbine within the farm. The notebook/script orchestrates a dialogue with the assistant, navigating through different inquiries and scenarios to demonstrate its capabilities.

### Data
This sample uses files from the folder [`data/energy/`](../data/energy/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb (hunk 7 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb:20]: # Personal Financial Assistant
## Objective

This notebook demonstrates the following:

- Function Calling with Yfinance to get latest stock prices. Summarization of user provided article. Extract country info from article, extract country, capital and other aspects, and call an API to get more information about each country.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant)
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 10-15 minutes running this sample. 

## About this example
The objective of the provided Python file is to create an Azure OpenAI Assistant named "Portfolio Management Assistant" using the Azure OpenAI API. The assistant is designed to act as a personal financial assistant, providing information and insights related to a user's investment portfolio. The script initiates a conversation with the assistant, guiding it through various financial queries and scenarios to showcase its capabilities.

### Data
This sample uses files from the folder [`data/finance/`](../data/finace/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb (hunk 8 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb:20]: # Wind Turbine Farm Management Assistant

## Objective
This notebook demonstrates the following:

- Utilizing Assistant tools such as the Code Interpreter and Function calling, this bot is capable of retrieving a CSV file that illustrates turbine wind speed, voltage, and the last maintenance date. It assists you in reviewing the file contents and aids in determining whether a specific turbine is in need of maintenance.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 5-10 minutes running this sample. 

## About this example
This sample demonstrates the creation of an Azure OpenAI Assistant named "Turbine Management Assistant" utilizing the Azure OpenAI API. The assistant functions as a management tool for wind turbine farms, providing data, charts, and actionable insights regarding the status and performance of each turbine within the farm. The notebook/script orchestrates a dialogue with the assistant, navigating through different inquiries and scenarios to demonstrate its capabilities.

### Data
This sample uses files from the folder [`data/energy/`](../data/energy/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb (hunk 5 above)
Comment (AndresPad) [reply scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb:37]: ## About this example

This sample shows users how to create an Azure OpenAI Assistant named "Math Tutor" using the Azure OpenAI API. The assistant is designed to function as a personal math tutor, capable of answering math questions through code interpretation. The script initiates a conversation with the assistant, guiding it through various mathematical queries and scenarios to showcase its capabilities.

This sample provides developers with a clear demonstration of how to leverage the core concepts of the Assistants API into their projects, highlighting its simplicity and effectiveness in leveraging foundational concepts.

**Comment by AndresPad** [comment]
@kdestin I just added in all the objectives and descriptions for the 4 notebooks. Let's sync up on the next set of changes.
thank you - Andres

This pull request includes several changes to improve the organization and clarity of the `scenarios/Assistants/api-in-a-box` project. The changes mainly focus on restructuring the README files, renaming and reorganizing the Jupyter notebooks, and modifying the code to enhance readability and maintainability.

Here are the most significant changes:

File and Folder Renaming:

* `scenarios/Assistants/assistants-api-in-a-box/README.md` was renamed to `scenarios/Assistants/api-in-a-box/README.md`. The contents of the file were also updated to reflect the new file paths.

Codebase Organization and Clarity:

* [`scenarios/Assistants/api-in-a-box/README.md`](diffhunk://#diff-11f4817e750bd51ec17268e6b689c6f9814fe83a16c7343022cbc4b736a4a0f2L142-R149): Fixed a typo in the description of the notebooks folder.
* [`scenarios/Assistants/api-in-a-box/README.md`](diffhunk://#diff-11f4817e750bd51ec17268e6b689c6f9814fe83a16c7343022cbc4b736a4a0f2L162-R162): Updated the paths to the Azure Open AI credentials in the `.env` file.
* [`scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb`](diffhunk://#diff-23cd1b85c2d4f1cc2f38ad18d6826f976b47fc7bf28fed0321f103569e0f9c29L7-R57): Reorganized the code to load environment variables and install packages at the beginning of the notebook. [[1]](diffhunk://#diff-23cd1b85c2d4f1cc2f38ad18d6826f976b47fc7bf28fed0321f103569e0f9c29L7-R57) [[2]](diffhunk://#diff-23cd1b85c2d4f1cc2f38ad18d6826f976b47fc7bf28fed0321f103569e0f9c29L18-R74)
* [`scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb`](diffhunk://#diff-a35c16932f8af523bbc9eac0b719797d6ddebaaf76329dfe2f002e5d2632d947L7-R63): Reorganized the code to load environment variables and install packages at the beginning of the notebook. [[1]](diffhunk://#diff-a35c16932f8af523bbc9eac0b719797d6ddebaaf76329dfe2f002e5d2632d947L7-R63) [[2]](diffhunk://#diff-a35c16932f8af523bbc9eac0b719797d6ddebaaf76329dfe2f002e5d2632d947L24-R80)

Documentation and Readability:

* [`scenarios/Assistants/api-in-a-box/failed_banks/README.md`](diffhunk://#diff-4e61504c7e920646de006f0bda06b33eef95298e0e69b42d1b59b27b1a407a67R1-R15): Added a new README file for the Failed Banks Information Assistant.
* [`scenarios/Assistants/api-in-a-box/math_tutor/README.md`](diffhunk://#diff-ad8cb9d5a6ec3af349b48827e966897e45217f0ae33afe1f6bc6d4e2b316d33fR1-R15): Added a new README file for the Math Tutor Assistant.
* [`scenarios/Assistants/api-in-a-box/personal_finance/README.md`](diffhunk://#diff-65bce0a64d889f7b9e69f258a358275b071bde76cccf759a746eaa32fad831e5R1-R15): Added a new README file for the Personal Financial Assistant.

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb (hunk 1 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/1e8d90231a99d4fc567c73b26279ba4a8484a754

File: scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb (hunk 2 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/6a54bf81f9d76b1c57a24badae1fba766e5cba67

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb (hunk 3 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/ca2359e0ec502e2143942885df0ad0c2ebcbfa47

Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/9a6b45d1e81a272bfed87e728d2b40ece73ce97e

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb (hunk 4 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/c98eaac6cf98fb54da5eb00215481f3fa0e2d6cb

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb (hunk 6 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/6833b0fb2ec9b571ea2f9fd191cde16e56bf4bbc

File: scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb (hunk 5 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/6118b1a20eeae2209d8b9442306d9a769b55096a

File: scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb (hunk 7 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/ca2359e0ec502e2143942885df0ad0c2ebcbfa47

File: scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb (hunk 8 above)
Comment (kdestin) [reply scenarios/Assistants/api-in-a-box/wind_farm/assistant-wind_farm.ipynb]: Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/b6d9e7d18f737a3812ef142e038bb4dd89e187ae

**Review by diondrapeck — APPROVED** [review]

**Review by AndresPad — APPROVED** [review]
@kdestin - just went through the changes and looks good. Once we get these changes in I will submit another PR to incorporate the new demo that we created.

**Pull request merged**